subworkflows/nf-core/fasta_newick_epang_gappa/tests/main.nf.test

```diff
@@ -103,6 +103,43 @@ nextflow_workflow {
             )
         }
     }
+
+    test("test_fasta_newick_epang_gappa_clustalo") {
+    
+        when {
+            workflow {
+                """
+                input[0] = Channel.of(
+                    [
+                        meta: [ id: "clustalo" ],
+                        data: [
+                            alignmethod:  'clustalo',
+                            queryseqfile: file("https://github.com/nf-core/test-datasets/raw/phyloplace/testdata/PF14720_3_sequences.faa", checkIfExists: true),
+                            refseqfile:   file("https://github.com/nf-core/test-datasets/raw/phyloplace/testdata/PF14720_seed.alnfaa", checkIfExists: true),
+                            refphylogeny: file("https://github.com/nf-core/test-datasets/raw/phyloplace/testdata/PF14720_seed.ft.LGCAT.newick", checkIfExists: true),
+                            model:        "LG",
+                            taxonomy:     file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/gappa/gappa_taxonomy.tsv", checkIfExists: true)
+                        ]
+                    ]
+                )
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(file(workflow.out.jplace.get(0).get(1)).name).match("clustalo.epa_result.jplace.gz") },
+                { assert snapshot(file(workflow.out.grafted_phylogeny.get(0).get(1)).name).match("clustalo.graft.placement.epa_result.newick") },
+                { assert snapshot(workflow.out.grafted_phylogeny.get(0).get(1)).md5().match("clustalo.graft.placement.epa_result.newick_lines") },
+                { assert snapshot(file(workflow.out.taxonomy_profile.get(0).get(1)).name).match("clustalo.taxonomy.profile.tsv") },
+                { assert snapshot(file(workflow.out.taxonomy_per_query.get(0).get(1)).name).match("clustalo.taxonomy.per_query.tsv") },
+                { assert snapshot(file(workflow.out.versions.get(0)).name).match("clustalo.versions.yml") },
+                { assert snapshot(workflow.out.versions.get(0)).md5().match("clustalo.versions.yml_lines") },
```

Comment:
This could probably be:
```suggestion
                { assert snapshot(
                  file(workflow.out.jplace.get(0).get(1)).name),
                  workflow.out.grafted_phylogeny.get(0).get(1)).md5(),
                  file(workflow.out.taxonomy_profile.get(0).get(1)).name),
                  file(workflow.out.taxonomy_per_query.get(0).get(1)).name),
                  file(workflow.out.versions.get(0)).name),
                  workflow.out.versions.get(0)).md5() 
                )
                },
```
(you don't need to check both the name and the md5sum for the grafted_phylogeny)